dkg/pedersen/dkg.go

```diff
@@ -174,7 +184,7 @@ func (o *Setup) allStartDeal(ssd structStartDeal) error {
 	var err error
 	if o.NewDKG == nil {
 		// TODO: Should be newdistkeyhandler
```

Comment:
What about this todo ?

Reply:
Not too sure, `NewDistKeyGenerator` appears to be just a convenience method for `NewDistKeyHandler`. There no sign that `NewDistKeyGenerator` is going away so I'll remove the TODO.